minigunapp/views.py

```diff
@@ -0,0 +1,36 @@
+from django.shortcuts import render
+from django.http import HttpResponse, Http404, JsonResponse
+from django.db import models
+
+import json
+import yaml
+from cerberus import Validator
+from minigunapp.models import Email
+from minigunapp.tasks import send_email
+
+v = Validator()
+schema = yaml.load(open('minigunapp/schemas/email.yaml'))
+
+def email(request):
+    if request.method in ['GET', 'POST']:
+        return globals()[request.method.lower() + '_email'](request) # WTB a real router
```

Comment:
It's smart but is it really necessary? What if you change the names of methods you pass the request to and forget to update this method?